config/locales/en.yml

```
@@ -276,10 +276,15 @@ en:
         about_accrediting_provider:
           text: "Enter details about the accredited provider"
   value_not_entered: "Not entered"
+  a_level_grades:
+    minimum_grade: "Grade %{minimum_grade} or above"
```

```suggestion
    minimum_grade: Grade %{minimum_grade} or above
```

Not a big deal, but you only need this with funny characters (like the max_grade below). The interpolation doesn't need quotes.

---

True. I am so confused in the locale. Some has quotes other doesn't.